modules/ROOT/pages/troubleshooting-issues-with-branded-emails.adoc

```diff
@@ -0,0 +1,89 @@
+= Issues with Branded Emails 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+
+ Anypoint API Community Manager sends notification emails when an API contract is approved or revoked. By default, these emails use Anypoint platform addresses and branding.
+ 
+ The following information includes symptoms and instructions for troubleshooting branded emails.
+
+== Data Capture Process Fails
+
+When the change data capture is enabled, Salesforce starts polling the ACM Community Bridge for contract updates. If the data capture process job fails, Salesforce stops the polling. 
+
+Check the status of the data capture process to confirm that the change data capture failed.
+
+To check the status, execute the following SOQL in the Developer Console:
+
+----
+SELECT Id, Name, SubmittedAt, StartedAt, FinishedAt, ProcessAfter, Status, Error
+FROM BackgroundOperation WHERE Name = 'acm_pkg__AnypointContracts__x' ORDER by SubmittedAt DESC LIMIT 15
+----
+
+To resolve this error, the administrator should resume the data capture process.
+
+== Not Receiving Emails
+
+API Community Manager sends notification emails when an API contract is approved or revoked. A common issue occurs when the user who set up the change data capture is not an organization administrator. The user must be an organizational administrator of the master organization to view all available applications and contracts in the organization. It is possible to receive no errors using SOQL since the job step could end successfully, but contract updates are not returned. 
+
+The first step of the change data capture job is to retrieve all available contracts with updates. The trigger that is set as part of the branded emails configuration runs if there are contract updates. If the trigger is not executed by the same user as the change data capture, Salesforce uses a user named Automated Process. This user is created in Anypoint the first time that the user tries to use the ACM Community Bridge. As result, you do not receive email notifications.
```

Comment:
```suggestion
The first step of the change data capture job is to retrieve all available contracts with updates. The trigger that is set as part of the branded emails configuration runs if there are contract updates. If the trigger is not executed by the same user as the change data capture, Salesforce uses a user named Automated Process. This user is created in Anypoint Platform the first time that the user tries to use the ACM Community Bridge. As result, you do not receive email notifications.
```

Reply:
There is a POV of shift in the last sentence when, it goes from user > you. Recommend revising to use "you" where possible in place of "user"